.cirrus.yml

```diff
@@ -31,6 +31,8 @@ openeuler_task:
     cpu: 4
     memory: 12G
   script: uname -a
-  setup_script: tests/ci/prepare-ci-environment.sh
+  setup_script: |
+    dnf install -y dnf-plugins-core
```

Comment:
This should happen in `tests/ci/prepare-ci-environment.sh`. I forgot it there.

Reply:
Moved it!